packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -329,9 +330,16 @@ export default class Tooltip extends PureComponent<TooltipProps> {
       return this.renderTooltipContent();
     }
 
+    let moveAmount : MoveBy = this.props.moveBy;
+
+    if (position && (position.includes("top") || position.includes("bottom"))) {
+        moveAmount = { main: 11, secondary: 0 };
```

Comment:
How will it apply to smaller components/elements? I feel like it will distant the tooltip from the referencing element